commerce_coordinator/apps/lms/views.py

```diff
@@ -82,10 +59,12 @@ def _redirect_response_payment(self, request):
         """
 
         get_items = list(self.request.GET.items())
-        get_items.append((f"{QueryParamPrefixes.WAFFLE_FLAG.value}{WaffleFlagNames.COORDINATOR_ENABLED.value}", '1'))
+        get_items.append((f"{QueryParamPrefixes.GOOGLE_ANALYTICS.value}", '1'))
```

Comment:
I still don't get why do we need google analytics query param in our new CC redirect URL as I previously mentioned. Nor I could find any such information in the ticket. Can you please mention the need for it?